### DISCUSSION TITLE
Not Found

### DISCUSSION THREAD

**Discussion by chwaki-a1** [Budibase/budibase#3307] [Help] [answered]
![image](https://user-images.githubusercontent.com/54703385/141093227-14841e95-a2f7-4e20-adff-a557796aec36.png)

What might be the problem? i am  stuck

**Comment by shogunpurple** [comment, marked as answer]
@chwaki27 

I'm not too sure what is happening here. Could you please add some more information?

It looks like you have tried to run budibase self hosted, and are attempting to access the builder on `localhost`.

`3188` is not the correct port when running self-hosted. You need to use `localhost:10000`.

If you provide some more info, about how you are trying to run budibase, I can help you.

**Reply by chwaki-a1** [reply]
Hello, Thanks.


Yes i'm trying to run Budibase self hosted through the configurations provided

![image](https://user-images.githubusercontent.com/54703385/141100367-7a4db4b3-f35d-41e1-ab52-59d4163540b6.png)

But the error is not found as below:
![image](https://user-images.githubusercontent.com/54703385/141100253-69804dd5-0a7a-4864-9495-6710e42dba02.png)

![image](https://user-images.githubusercontent.com/54703385/141100570-4590e9dc-1858-4305-b8eb-d4cb1c3a0891.png)

![image](https://user-images.githubusercontent.com/54703385/141100725-725f72aa-981b-4da8-a1c1-e90d12bd39e9.png)

**Reply by shogunpurple** [reply]
@chwaki27 could you try clearing your cookies in your browser? Everything there in your configuration looks fine, provided that you just followed the budibase CLI instructions.

**Reply by chwaki-a1** [reply]
@shogunpurple Thank you Loads,

After clearing Cookie now it's fine

**Reply by shogunpurple** [reply]
Awesome @chwaki27, glad you are up and running!
